### PR TITLE
Update react.json

### LINKED ISSUE #490: CodeSandbox
### URL

https://codesandbox.io/search?refinementList

### Description of the above provided site

This site will give you a every type of design with code. From this you can make your coding journey very fast.

### Category

Frontend

### Sub Category

CAT = FRONTEND
SUBCAT = REACTJS

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #490 

## Changes proposed

It will give you more design and make your coding journey very fast.

## Screenshots

Screenshot (790)
Screenshot (791)

## Note to reviewers

Any kind of feedback would be appreciated

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
         "category":"frontend",
         "subcategory":"react",
         "language":"english"
+    },
+    {
+        "name":"CodeSandbox",
+        "description":"This site will give you a every type of design with code. From this you can make your coding journey very fast.",
```
Comment (rupali-codes) [inline database/frontend/react.json:12]: Could make the description a bit larger? it looks so small

**Review by parma1111 — COMMENTED** [review]
everything is ok

File: database/frontend/react.json [hunk 2]
```diff
@@ -6,5 +6,13 @@
         "category":"frontend",
         "subcategory":"react",
         "language":"english"
+    },
+    {
+        "name":"CodeSandbox",
+        "description":"This site will give you a every type of design with code. From this you can make your coding journey very fast.You can filter your design by framework,dependiences and tags.",
```
Comment (CBID2) [inline database/frontend/react.json:12]:
```suggestion
        "description":"This IDE will make it easier for you to code. You can get any design you want as well as filter it by framework,dependiences, and tags.",
```

File: database/frontend/react.json [hunk 3]
```diff
@@ -6,5 +6,13 @@
         "category":"frontend",
         "subcategory":"react",
```
Comment (CBID2) [inline database/frontend/react.json:7]: I'd put it in the "Dev tools" category since it's an IDE.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Great start so far @parma1111! 😀  I just made some suggestions on how you can improve it.

**Comment by rupali-codes** [comment]
@CBID2 could you review the PR once again 😅

**Review by rupali-codes — APPROVED** [review]
looks good to me too

**Pull request merged**